sympy/solvers/ode.py

```diff
@@ -2955,8 +2966,17 @@ def _constant_renumber(expr):
             sortedargs.sort(key=sort_key)
             return expr.func(*[_constant_renumber(x) for x in sortedargs])
     expr = _constant_renumber(expr)
+
+    # Use Symbols not in the original ODE
+    if exclude is None:
+        exclude = []
+    dummy_ode = Add(*exclude)
+    newconsts = get_numbered_constants(dummy_ode, num=newstartnumber)
+    # FIXME: Why does get_numbered_constants not always return a list?
```

Comment:
check the code: `return (Cs[0] if num == 1 else tuple(Cs))`. When `num == 1` you get a single constant. So you could use 
```python
if newstartnumber == 1:
    newconsts = [newconsts]
```

Reply:
or just use `newconsts = list(iter_numbered_constants(dummy_ode, num=newstartnumber))`.